src/main/java/org/hmhb/kml/DefaultKmlService.java

```diff
@@ -0,0 +1,285 @@
+package org.hmhb.kml;
+
+import javax.annotation.Nonnull;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.hmhb.county.County;
+import org.hmhb.program.Program;
+import org.hmhb.program.ProgramService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import static java.util.Objects.requireNonNull;
+
+@Service
+public class DefaultKmlService implements KmlService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultKmlService.class);
+
+    private final ProgramService programService;
+
+    @Autowired
+    public DefaultKmlService(
+            @Nonnull ProgramService programService
+    ) {
+        LOGGER.debug("constructed");
+        this.programService = requireNonNull(programService, "programService cannot be null");
+    }
+
+/*
+https://labs.micromata.de/projects/jak/quickstart.html
+
+https://labs.micromata.de/projects/jak.html
+
+https://github.com/micromata/javaapiforkml
+
+http://stackoverflow.com/questions/15039804/jak-no-longer-available-as-it-seems-create-kml-with-jaxb
+
+http://stackoverflow.com/questions/2952024/which-java-libraries-can-be-used-to-generate-kml
+
+https://labs.micromata.de/projects/jak/kml-in-the-java-world.html
+*/
+
+    private KmlRoot doSomething() {
```

Comment:
This hardcoded data isn't needed anymore now that I have the full integration working, so remove it.